composer.json

```diff
@@ -36,33 +36,32 @@
         "ext-json": "*",
         "ext-simplexml": "*",
         "composer-runtime-api": "^2.0",
-        "coduo/php-matcher": "^6.0.14",
         "composer/package-versions-deprecated": "^1.8",
         "contao/imagine-svg": "^1.0",
-        "doctrine/annotations": "^1.2 || ^2.0",
-        "doctrine/cache": "^1.0.1",
-        "doctrine/collections": "^1.0 || ^2.0",
-        "doctrine/data-fixtures": "^1.3.3",
-        "doctrine/dbal": "^2.13 || ^3.0",
-        "doctrine/doctrine-bundle": "^2.4",
-        "doctrine/doctrine-fixtures-bundle": "^3.3 || ^4.0",
+        "doctrine/annotations": "^1.14 || ^2.0",
+        "doctrine/cache": "^1.12.1",
+        "doctrine/collections": "^1.6 || ^2.0",
+        "doctrine/data-fixtures": "^1.4",
+        "doctrine/dbal": "^3.3",
+        "doctrine/doctrine-bundle": "^2.6",
+        "doctrine/doctrine-fixtures-bundle": "^3.4 || ^4.0",
         "doctrine/inflector": "^1.4.1 || ^2.0.1",
         "doctrine/orm": "^2.13",
-        "doctrine/persistence": "^2.0 || ^3.0",
-        "doctrine/phpcr-bundle": "^2.2 || ^3.0",
+        "doctrine/persistence": "^3.1",
+        "doctrine/phpcr-bundle": "^2.4 || ^3.0",
         "dragonmantank/cron-expression": "^1.1 || ^2.0 || ^3.0",
         "friendsofphp/proxy-manager-lts": "^1.0",
-        "friendsofsymfony/http-cache": "^2.10 || ^3.0",
-        "friendsofsymfony/http-cache-bundle": "^2.10.1 || ^3.0",
-        "friendsofsymfony/jsrouting-bundle": "^2.7 || ^3.0",
-        "friendsofsymfony/rest-bundle": "^3.1",
-        "gedmo/doctrine-extensions": "^3.0.4",
+        "friendsofsymfony/http-cache": "^2.13 || ^3.0",
+        "friendsofsymfony/http-cache-bundle": "^2.12 || ^3.0",
+        "friendsofsymfony/jsrouting-bundle": "^3.0",
+        "friendsofsymfony/rest-bundle": "^3.2",
+        "gedmo/doctrine-extensions": "^3.8",
```

Comment:
gedmo has dependency to doctrine persistence in older versions not defined and so runs into:

> Error: Call to undefined method Doctrine\Bundle\DoctrineBundle\Mapping\ClassMetadataFactory::getCacheDriver()

We need increase it this to atleast 3.5 or 3.6 but our lowest supported persistence versions already update it then to 3.8 so define ^3.8 here.